docsrc/sample.rst

```diff
@@ -58,39 +59,63 @@ All of these arguments are optional; when unspecified, the CmdStan defaults will
 See :meth:`~cmdstanpy.CmdStanModel.sample` for more details about the parameters.
 
 
-Example: fit model - sampler defaults
--------------------------------------
+CmdStanMCMC: HMC sample and metadata
+------------------------------------
 
-In this example we use the CmdStan example model
-`bernoulli.stan <https://github.com/stan-dev/cmdstanpy/blob/master/test/data/bernoulli.stan>`__
-and data file
-`bernoulli.data.json <https://github.com/stan-dev/cmdstanpy/blob/master/test/data/bernoulli.data.json>`__.
+The :ref:`class_cmdstanmodel` class method  ``sample`` returns a :ref:`class_cmdstanmcmc` object
+which provides properties and methods to access and manage the sample; these fall into the following
+following functional categories:
+
+**Get sample contents**
 
-The :ref:`class_cmdstanmodel` class method  ``sample`` returns a ``CmdStanMCMC`` object
-which provides properties to retrieve information about the sample, as well as methods
-to run CmdStan's summary and diagnostics tools.
+- ``draws`` - The numpy.ndarray which contains all across all chains arranged as (draws, chains, columns).
+- ``stan_variable(name=var_name)`` - Returns a numpy.ndarray which contains the set of draws in the sample for the named Stan program variable.
+- ``stan_variables()`` - Returns a Python dict, key: Stan program variable name, value: numpy.ndarray of draws.
 
-Methods for information about the fit of the model to the data:
+- ``metric`` - List of per-chain metric values, metric is either a vector ('diag_e') or matrix ('dense_e')
+- ``stepsize`` - List of per-chain step sizes.
+
+**Get sample metadata**
+
+- ``column_names`` - List of column labels for one draw from the sampler.
+- ``sampler_vars_cols`` - Python dict, key: sampler parameter name, value: zero-based output column index.
+- ``stan_vars_cols`` - Python dict, key: Stan program variable name, value: tuple of output column indices.
+- ``stan_vars_dims`` - Python dict, key: Stan program variable name, value: tuple of dimensions, or empty tuple, for scalar variables.
+
+- ``cmdstan_config`` - Python dict, key: CmdStan argument name, value: value used for this sampler run, whether user-specified or CmdStan default.
+  
+- ``chains`` - Number of chains 
+- ``thin`` - Period between recorded iterations.
+
+- ``num_draws`` - Number of draws per chain, i.e., thinned iterations.
+- ``num_draws_sampling`` - Number of sampling (post-warmup) draws per chain, i.e., sampling iterations, thinned.  By default, only the post-warmup draws are reported, so that ``num_draws`` == ``num_draws_sampling``.
```

Comment:
The `num_draws` behavior is a bug or this is wrong.

Reply:
fixed